test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
IIRC the model matrix code takes advantage of the categorical-ness of `CategoricalArray`s (in StatsModels) and `PooledDataArray`s (in the formula code here), so I don't think StatsModels could go fully against `AbstractArray`.